miqa/core/tests/pyppeteer/test_scan_upload.py

```diff
@@ -63,6 +63,6 @@ async def test_upload_scans(
     await scans[0].click()
 
     assert await page.waitForXPath(
-        '//div[contains(@class, "col")][contains(., "Demo Project")]'
-        '/following::div[contains(@class, "col")][contains(., "Test Experiment")]'
+        '//div[contains(@class, "d-flex")][contains(., "Demo Project")]'
```

Comment:
It's probably a better practice to put a custom class on this rather than strictly depending on a `d-flex`-classed element. E.g. `class="miqa-project-title"`

Reply:
Changed in [0204ae3](https://github.com/OpenImaging/miqa/pull/563/commits/0204ae3d19d755b3fe8e7acbd71355c81b7a63be)